release-notes/7.0/preview/api-diff/preview4/Microsoft.NETCore.App/7.0-preview4_System.md

```diff
@@ -0,0 +1,2179 @@
++        static Double System.Numerics.IDivisionOperators<System.Double,System.Double,System.Double>.op_CheckedDivision(Double left, Double right);
++        static Double System.Numerics.IDivisionOperators<System.Double,System.Double,System.Double>.op_Division(Double left, Double right);
++        int System.Numerics.IFloatingPoint<System.Double>.GetExponentByteCount();
++        long System.Numerics.IFloatingPoint<System.Double>.GetExponentShortestBitLength();
++        long System.Numerics.IFloatingPoint<System.Double>.GetSignificandBitLength();
++        int System.Numerics.IFloatingPoint<System.Double>.GetSignificandByteCount();
++        bool System.Numerics.IFloatingPoint<System.Double>.TryWriteExponentLittleEndian(Span<byte> destination, out int bytesWritten);
++        bool System.Numerics.IFloatingPoint<System.Double>.TryWriteSignificandLittleEndian(Span<byte> destination, out int bytesWritten);
++        static Double System.Numerics.IIncrementOperators<System.Double>.op_CheckedIncrement(Double value);
++        static Double System.Numerics.IIncrementOperators<System.Double>.op_Increment(Double value);
++        static Double System.Numerics.IModulusOperators<System.Double,System.Double,System.Double>.op_Modulus(Double left, Double right);
++        static Double System.Numerics.IMultiplyOperators<System.Double,System.Double,System.Double>.op_CheckedMultiply(Double left, Double right);
++        static Double System.Numerics.IMultiplyOperators<System.Double,System.Double,System.Double>.op_Multiply(Double left, Double right);
++        static Double System.Numerics.ISubtractionOperators<System.Double,System.Double,System.Double>.op_CheckedSubtraction(Double left, Double right);
++        static Double System.Numerics.ISubtractionOperators<System.Double,System.Double,System.Double>.op_Subtraction(Double left, Double right);
++        static Double System.Numerics.IUnaryNegationOperators<System.Double,System.Double>.op_CheckedUnaryNegation(Double value);
++        static Double System.Numerics.IUnaryNegationOperators<System.Double,System.Double>.op_UnaryNegation(Double value);
++        static Double System.Numerics.IUnaryPlusOperators<System.Double,System.Double>.op_UnaryPlus(Double value);
+     }
+     public static class GC {
++        public static GCNotificationStatus WaitForFullGCApproach(TimeSpan timeout);
++        public static GCNotificationStatus WaitForFullGCComplete(TimeSpan timeout);
+     }
+-    public readonly struct Guid : IComparable, IComparable<Guid>, IComparisonOperators<Guid, Guid>, IEqualityOperators<Guid, Guid>, IEquatable<Guid>, IFormattable, IParseable<Guid>, ISpanFormattable, ISpanParseable<Guid>
++    public readonly struct Guid : IComparable, IComparable<Guid>, IComparisonOperators<Guid, Guid>, IEqualityOperators<Guid, Guid>, IEquatable<Guid>, IFormattable, IParsable<Guid>, ISpanFormattable, ISpanParsable<Guid>
```

Comment:
As indicated above, several of the types don't implement the `generic math` operators in P5. It may be less confusing to just never include them in the `diff`.
```suggestion
+    public readonly struct Guid : IComparable, IComparable<Guid>, IEquatable<Guid>, IFormattable, IParsable<Guid>, ISpanFormattable, ISpanParsable<Guid>
```